exporters/stats/stackdriver/src/test/java/io/opencensus/exporter/stats/stackdriver/StackdriverStatsExporterTest.java

```diff
@@ -165,8 +165,9 @@ public void registerViewAndExport() throws IOException {
       StackdriverStatsExporter.registerView(view);
 
       verify(mockStub, times(1)).createMetricDescriptorCallable();
-      // The timeout for verifying createTimeSeries needs to match the export interval of exporter.
-      verify(mockStub, timeout(1000).times(1)).createTimeSeriesCallable();
+      // This verification is slower on some machine. Relax the timeout be greater than the export
+      // interval to make this more steady.
+      verify(mockStub, timeout(2000).times(1)).createTimeSeriesCallable();
```

Comment:
Shouldn't this test use `atLeast(1)` instead of `times(1)` when verifying `createTimeSeriesCallable()`, since the method could be called twice in the new time interval?

Reply:
Done. Actually `timeout()` only waits for the first invocation, so `atLeast(1)` and `times(1)` will be the same here.